src/app/create/codebases/codebases.module.ts

```diff
@@ -1,3 +1,4 @@
+import { module } from '@angular/upgrade/src/angular_js';
```

Comment:
I did not intend to; I did not type that. Perhaps some VS Code extension did it.